### PR TITLE
adding link to the dependency window doc

### PR DESCRIPTION
**Pull request by Yoyozilla**
## Changes
- Fixes: # .

### CODE REVIEW

File: Documentation/toc.yml [hunk 1]
```diff
@@ -140,6 +140,8 @@
     items:
     - name: Build Window
       href: Utilities/BuildWindow.md
+    - name: Build Window
+      href: DependencyWindow.md
```
Comment (Cameron-Micka) [inline Documentation/toc.yml]: I'm not too familiar with the "toc" file. Does this line need to be in the utilities namespace?
  
`href: Utilities/DependencyWindow.md`

File: Documentation/toc.yml [hunk 2]
```diff
@@ -140,6 +140,8 @@
     items:
     - name: Build Window
       href: Utilities/BuildWindow.md
+    - name: Build Window
```
Comment (Cameron-Micka) [inline Documentation/toc.yml]: Should this be?

`- name: Dependency Window`

File: Documentation/toc.yml (hunk 1 above)
Comment (Yoyozilla) [reply Documentation/toc.yml]: Checking..

File: Documentation/toc.yml (hunk 2 above)
Comment (Yoyozilla) [reply Documentation/toc.yml]: fixed

**Pull request merged**